components/login/index.test.jsx

```diff
@@ -118,4 +119,27 @@ describe("Login form", () => {
     expect(loginArgs.oidcIssuer).toBe("https://login.inrupt.com");
     expect(loginArgs.redirectUrl).toBe("http://localhost:3000/");
   });
+
+  // FIXME: remove once pod migration is completed
+  it("clicking the Sign In button does not login and displays error message", async () => {
+    const session = mockUnauthenticatedSession();
+    const SessionProvider = mockSessionContextProvider(session, false, null);
+
+    const { asFragment, getByTestId } = renderWithTheme(
+      <SessionProvider>
+        <Login />
+      </SessionProvider>
+    );
+
+    const signinButton = getByTestId(TESTCAFE_ID_LOGIN_BUTTON);
+
+    act(() => {
+      signinButton.click();
+    });
+
+    expect(mockUseReturnUrl.persist).not.toHaveBeenCalled();
+    expect(session.login).not.toHaveBeenCalled();
+
+    expect(asFragment).toMatchSnapshot();
+  });
```

Comment:
Maybe add a test case proving that CSS/NSS could be used? idk.

Reply:
Added test - passed an IDP value via query params to save myself the whole clicking through the dropdown thing.